ddpui/core/dbt_automation/operations/mergeoperations.py

```
@@ -1,30 +1,30 @@
+from ddpui.core.dbt_automation.operations.mergetables import union_tables_sql
+from ddpui.core.dbt_automation.operations.regexextraction import regex_extraction_sql
+from ddpui.core.dbt_automation.utils.dbtproject import dbtProject
+from ddpui.utils.warehouse.old_client.warehouse_interface import WarehouseInterface
+from ddpui.core.dbt_automation.operations.castdatatypes import cast_datatypes_sql
+from ddpui.core.dbt_automation.operations.replace import replace_dbt_sql
+from ddpui.core.dbt_automation.operations.joins import joins_sql
+from ddpui.core.dbt_automation.operations.wherefilter import where_filter_sql
+from ddpui.core.dbt_automation.operations.groupby import groupby_dbt_sql
+from ddpui.core.dbt_automation.operations.aggregate import aggregate_dbt_sql
+from ddpui.core.dbt_automation.operations.casewhen import casewhen_dbt_sql
+from ddpui.core.dbt_automation.operations.flattenjson import flattenjson_dbt_sql
```

Ruff (0.15.1)

[error] Redefinition of unused `flattenjson_dbt_sql` from line 10: `flattenjson_dbt_sql` redefined here

Remove definition: `flattenjson_dbt_sql`

(F811)